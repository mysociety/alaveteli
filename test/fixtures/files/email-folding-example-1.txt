Dear Mr Pollard,

Thank you for your email of 26 February. Please find a response attached.

Yours faithfully,

On behalf of James Hall, Chief Executive
Identity and Passport Service

<<9032 C Pollard final response.doc>>

**********************************************************************

This email and any files transmitted with it are confidential and

intended solely for the use of the individual or entity to whom they

are addressed. If you have received this email in error please notify

the system manager.

This footnote also confirms that this email message has been swept for the
presence of computer viruses.

**********************************************************************

The original of this email was scanned for viruses by the Government
Secure Intranet virus scanning service supplied by Cable&Wireless in
partnership with MessageLabs. (CCTM Certificate Number 2007/11/0032.) On
leaving the GSi this email was certified virus free.
Communications via the GSi may be automatically logged, monitored and/or
recorded for legal purposes.
