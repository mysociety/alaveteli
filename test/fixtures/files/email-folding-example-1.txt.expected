Dear Mr Pollard,

Thank you for your email of 26 February. Please find a response attached.

Yours faithfully,

On behalf of James Hall, Chief Executive
Identity and Passport Service

<<9032 C Pollard final response.doc>>
